ClassAssignment.md

```diff
@@ -14,6 +14,6 @@ For this assignment please answer the following questions in a [Public GitHub Gi
 
 ####Please add your Gist links here in the following format:
 
-- Name:
-- Email:
-- Link:
+- Name: Gary Mouradian
+- Email: gary.c.mouradian@gmail.com
+- Link: [MyAssignment](https://gist.github.com/GaryMouradian/e314114eab88de159a34de3ab58f5d55)
```

Comment:
Hi Gary! Thanks again for coming. I am leaving you a GitHub comment, which is how developers usually communicate regarding Pull Requests (proposed changes to code).

So, I realized if you put a link as just a string of text starting with "http://..." without the [Link Text](Link Address) syntax, GitHub's editor is smart enough to turn it into a hyperlink.

Still, if you want to write custom link text on GitHub, you have to use the "[Link Text](Link Address)" syntax.